src/containers/BooksList.js

```diff
@@ -21,14 +21,7 @@ const BooksList = () => {
   return (
     <>
       <CategoryFilter target={target} handleFilter={handleFilter} />
```

Comment:
Let's reposition the filter component to look like the given design.

I will like to show you what that means according to the [project design](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376032/);

Project Design: So once you click categories, the dropdown should appear.

![image](https://user-images.githubusercontent.com/31897434/128144850-4eb83c5b-3ac1-4afd-a767-8b6ee7055458.png)

Another similar example: Here we just need to place the dropdown beside the text `Books`

![image](https://user-images.githubusercontent.com/31897434/128145148-7818df9d-9c04-4720-be7a-a5fc88702ddb.png)


Your Design:

![image](https://user-images.githubusercontent.com/31897434/130485475-5d9d765f-bf78-427a-a326-5c366b1db660.png)



I hope it is more clearer now, kindly fix this 👍 .

Reply:
but this is not in the design provided